src/js/profile/mobile/widget/Calendar.js

```diff
@@ -70,12 +101,21 @@
 			*/
 			prototype._init = function (element) {
 				var self = this,
-					calendarElement = self.element.querySelector(".calendarView");
+					calendarElement = self.element.querySelector("." + Calendar.classes.CALENDAR_VIEW);
```

Comment:
use in this place `classes` instead of `Calendar.classes`

Reply:
done.